#! /bin/sh /usr/share/dpatch/dpatch-run
## epsg-codes-loc.dpatch by  <debian@adkgis.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: look for epsg codes in the right place

@DPATCH@
diff -urNad grass-5.7.0+6.0.0beta1/lib/init/epsg_option.tcl /tmp/dpep.Ly5J8O/grass-5.7.0+6.0.0beta1/lib/init/epsg_option.tcl
--- grass-5.7.0+6.0.0beta1/lib/init/epsg_option.tcl	2005-01-12 09:01:55.000000000 -0500
+++ /tmp/dpep.Ly5J8O/grass-5.7.0+6.0.0beta1/lib/init/epsg_option.tcl	2005-01-19 11:06:33.000000000 -0500
@@ -58,7 +58,7 @@
         global thelocation
         global browsedepsg 
         
-        set browsedepsg "/usr/local/share/proj/epsg"
+        set browsedepsg "/usr/share/proj/epsg"
         set epsgLocation "newLocation"
         set epsg_code ""
         
@@ -91,7 +91,7 @@
                                         
         bind .optPopup.input3_ppbEntry <Leave> {
              if {$browsedepsg == ""} {
-                set browsedepsg "/usr/local/share/proj/epsg"
+                set browsedepsg "/usr/share/proj/epsg"
             }
         }
 
@@ -270,7 +270,7 @@
         .infoPopup.text insert end "\nThe Grass database in which the location should be created\n\n"
         .infoPopup.text insert end "\n" 
         .infoPopup.text insert end "\n   EPSG code number of projection:\n\n" subtitle
-        .infoPopup.text insert end "\nEPSG code number of projection (see /usr/local/share/proj/epsg or push the 'EPSG-codes' button)\n\n"
+        .infoPopup.text insert end "\nEPSG code number of projection (see /usr/share/proj/epsg or push the 'EPSG-codes' button)\n\n"
         
         pack .infoPopup.text -side left -fill both 
         
